src/SUMMARY.md

```diff
@@ -39,9 +39,19 @@
    - [Packages](./maintenance/packages/index.md)
       - [General](./maintenance/packages/management.md)
       - [Building](./maintenance/packages/building.md)
-      - [Debugging](./maintenance/packages/debugging.md)
-      - [Mirrors](./maintenance/packages/mirrors.md)
       - [Finding files](./maintenance/packages/files.md)
+   - [Repositories](./maintenance/repositories/index.md)
+      - [Official Repositories](./maintenance/repositories/official/index.md)
+         - [Installing
```

Comment:
Is this formatted correctly?

Reply:
Is it because of the 80 character limit? If so, perhaps we could change the limit for this particular file, since it's full of single-lines anyways, and more than likely we'll get more lines that exceed the 80 characters in this file in the future.